[
    {
        "user": "U4H19D1D2",
        "type": "message",
        "ts": "1591877997.063400",
        "edited": {
            "user": "U4H19D1D2",
            "ts": "1591878086.000000"
        },
        "client_msg_id": "c949780b-705f-4fa8-a56a-eb6f7652da8f",
        "text": "meltdown? <https:\/\/status.developer.betfair.com\/>",
        "team": "T4G9NBD2M",
        "user_team": "T4G9NBD2M",
        "source_team": "T4G9NBD2M",
        "user_profile": {
            "avatar_hash": "137c5a3ef323",
            "image_72": "https:\/\/avatars.slack-edge.com\/2024-03-15\/6819395173841_137c5a3ef323f1944a1a_72.png",
            "first_name": "liam",
            "real_name": "liam",
            "display_name": "liam",
            "team": "T4G9NBD2M",
            "name": "liam",
            "is_restricted": false,
            "is_ultra_restricted": false
        },
        "thread_ts": "1591877997.063400",
        "reply_count": 1,
        "reply_users_count": 1,
        "latest_reply": "1591878206.064100",
        "reply_users": [
            "U010GM77S4W"
        ],
        "replies": [
            {
                "user": "U010GM77S4W",
                "ts": "1591878206.064100"
            }
        ],
        "is_locked": false,
        "subscribed": true,
        "last_read": "1591878206.064100",
        "blocks": [
            {
                "type": "rich_text",
                "block_id": "aH+HB",
                "elements": [
                    {
                        "type": "rich_text_section",
                        "elements": [
                            {
                                "type": "text",
                                "text": "meltdown? "
                            },
                            {
                                "type": "link",
                                "url": "https:\/\/status.developer.betfair.com\/"
                            }
                        ]
                    }
                ]
            }
        ],
        "reactions": [
            {
                "name": "+1",
                "users": [
                    "UU1URJ8L8",
                    "U92CASP1B"
                ],
                "count": 2
            }
        ]
    },
    {
        "user": "U010GM77S4W",
        "type": "message",
        "ts": "1591878206.064100",
        "client_msg_id": "833c76b5-6048-4932-b960-dd588a22c29b",
        "text": "yes, fingers crossed",
        "team": "T4G9NBD2M",
        "user_team": "T4G9NBD2M",
        "source_team": "T4G9NBD2M",
        "user_profile": {
            "avatar_hash": "66cae8f5399c",
            "image_72": "https:\/\/avatars.slack-edge.com\/2020-03-20\/1004508356769_66cae8f5399cf91a07ae_72.jpg",
            "first_name": "",
            "real_name": "mandelbot",
            "display_name": "mandelbot",
            "team": "T4G9NBD2M",
            "name": "demhadbale",
            "is_restricted": false,
            "is_ultra_restricted": false
        },
        "thread_ts": "1591877997.063400",
        "parent_user_id": "U4H19D1D2",
        "blocks": [
            {
                "type": "rich_text",
                "block_id": "v29",
                "elements": [
                    {
                        "type": "rich_text_section",
                        "elements": [
                            {
                                "type": "text",
                                "text": "yes, fingers crossed"
                            }
                        ]
                    }
                ]
            }
        ],
        "reactions": [
            {
                "name": "+1",
                "users": [
                    "U92CASP1B"
                ],
                "count": 1
            }
        ]
    },
    {
        "user": "U0131E2H1SP",
        "type": "message",
        "ts": "1591878227.064400",
        "client_msg_id": "12477b41-19ae-4940-a3bd-1e9fc4f8d0c9",
        "text": "Yep…. this is the first place i came to check",
        "team": "T4G9NBD2M",
        "user_team": "T4G9NBD2M",
        "source_team": "T4G9NBD2M",
        "user_profile": {
            "avatar_hash": "g78ded1f7356",
            "image_72": "https:\/\/secure.gravatar.com\/avatar\/78ded1f7356527198fa2768d88bfd18b.jpg?s=72&d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0001-72.png",
            "first_name": "",
            "real_name": "Dave Simonds",
            "display_name": "",
            "team": "T4G9NBD2M",
            "name": "davejsimonds",
            "is_restricted": false,
            "is_ultra_restricted": false
        },
        "blocks": [
            {
                "type": "rich_text",
                "block_id": "OIjh",
                "elements": [
                    {
                        "type": "rich_text_section",
                        "elements": [
                            {
                                "type": "text",
                                "text": "Yep…. this is the first place i came to check"
                            }
                        ]
                    }
                ]
            }
        ]
    },
    {
        "user": "U92CASP1B",
        "type": "message",
        "ts": "1591879025.064900",
        "client_msg_id": "71f8ce6f-6060-4dc2-b6fb-0760322edacc",
        "text": "At least they are sending heartbeats ...",
        "team": "T4G9NBD2M",
        "user_team": "T4G9NBD2M",
        "source_team": "T4G9NBD2M",
        "user_profile": {
            "avatar_hash": "g374352c822f",
            "image_72": "https:\/\/secure.gravatar.com\/avatar\/a374352c822fa856c4cfdd782bbb8669.jpg?s=72&d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0007-72.png",
            "first_name": "Jonatan",
            "real_name": "Jonatan",
            "display_name": "Jonatan (skyw)",
            "team": "T4G9NBD2M",
            "name": "almen.jonatan",
            "is_restricted": false,
            "is_ultra_restricted": false
        },
        "blocks": [
            {
                "type": "rich_text",
                "block_id": "iswdt",
                "elements": [
                    {
                        "type": "rich_text_section",
                        "elements": [
                            {
                                "type": "text",
                                "text": "At least they are sending heartbeats ..."
                            }
                        ]
                    }
                ]
            }
        ]
    },
    {
        "subtype": "channel_join",
        "user": "U0151JA96F8",
        "text": "<@U0151JA96F8> has joined the channel",
        "type": "message",
        "ts": "1591880033.065400"
    },
    {
        "user": "UN1497LJC",
        "type": "message",
        "ts": "1591880353.065900",
        "client_msg_id": "3cb4c527-e114-470d-a998-56d93f9a4a25",
        "text": "Oh dear. I think they might have been doing some changes, as my account page changed again this morning",
        "team": "T4G9NBD2M",
        "user_team": "T4G9NBD2M",
        "source_team": "T4G9NBD2M",
        "user_profile": {
            "avatar_hash": "ga740a9a5a57",
            "image_72": "https:\/\/secure.gravatar.com\/avatar\/1a740a9a5a57633755157e17ebe96b27.jpg?s=72&d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0018-72.png",
            "first_name": "",
            "real_name": "JonJonJon",
            "display_name": "JonJonJon",
            "team": "T4G9NBD2M",
            "name": "edgetrading2",
            "is_restricted": false,
            "is_ultra_restricted": false
        },
        "blocks": [
            {
                "type": "rich_text",
                "block_id": "D9cq",
                "elements": [
                    {
                        "type": "rich_text_section",
                        "elements": [
                            {
                                "type": "text",
                                "text": "Oh dear. I think they might have been doing some changes, as my account page changed again this morning"
                            }
                        ]
                    }
                ]
            }
        ]
    },
    {
        "user": "U010GM77S4W",
        "type": "message",
        "ts": "1591880421.066300",
        "client_msg_id": "39a86a6e-cac6-4b9d-bed6-5f4a7952c0c1",
        "text": "They've been suspending events when they do changes. I think this is a crash.",
        "team": "T4G9NBD2M",
        "user_team": "T4G9NBD2M",
        "source_team": "T4G9NBD2M",
        "user_profile": {
            "avatar_hash": "66cae8f5399c",
            "image_72": "https:\/\/avatars.slack-edge.com\/2020-03-20\/1004508356769_66cae8f5399cf91a07ae_72.jpg",
            "first_name": "",
            "real_name": "mandelbot",
            "display_name": "mandelbot",
            "team": "T4G9NBD2M",
            "name": "demhadbale",
            "is_restricted": false,
            "is_ultra_restricted": false
        },
        "blocks": [
            {
                "type": "rich_text",
                "block_id": "dzy",
                "elements": [
                    {
                        "type": "rich_text_section",
                        "elements": [
                            {
                                "type": "text",
                                "text": "They've been suspending events when they do changes. I think this is a crash."
                            }
                        ]
                    }
                ]
            }
        ]
    },
    {
        "user": "U92CASP1B",
        "type": "message",
        "ts": "1591880907.066900",
        "client_msg_id": "7900c637-a37a-4e0d-b29d-986f1f697dfc",
        "text": "Yeah, looks like everything is about to go down now",
        "team": "T4G9NBD2M",
        "user_team": "T4G9NBD2M",
        "source_team": "T4G9NBD2M",
        "user_profile": {
            "avatar_hash": "g374352c822f",
            "image_72": "https:\/\/secure.gravatar.com\/avatar\/a374352c822fa856c4cfdd782bbb8669.jpg?s=72&d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0007-72.png",
            "first_name": "Jonatan",
            "real_name": "Jonatan",
            "display_name": "Jonatan (skyw)",
            "team": "T4G9NBD2M",
            "name": "almen.jonatan",
            "is_restricted": false,
            "is_ultra_restricted": false
        },
        "blocks": [
            {
                "type": "rich_text",
                "block_id": "QMfDg",
                "elements": [
                    {
                        "type": "rich_text_section",
                        "elements": [
                            {
                                "type": "text",
                                "text": "Yeah, looks like everything is about to go down now"
                            }
                        ]
                    }
                ]
            }
        ]
    },
    {
        "user": "U4H19D1D2",
        "type": "message",
        "ts": "1591881390.067900",
        "client_msg_id": "3c1e3c3c-234d-498c-8a53-3c5b7f88a0cf",
        "text": "yeah this is a bad one, tbf its been a while",
        "team": "T4G9NBD2M",
        "user_team": "T4G9NBD2M",
        "source_team": "T4G9NBD2M",
        "user_profile": {
            "avatar_hash": "137c5a3ef323",
            "image_72": "https:\/\/avatars.slack-edge.com\/2024-03-15\/6819395173841_137c5a3ef323f1944a1a_72.png",
            "first_name": "liam",
            "real_name": "liam",
            "display_name": "liam",
            "team": "T4G9NBD2M",
            "name": "liam",
            "is_restricted": false,
            "is_ultra_restricted": false
        },
        "blocks": [
            {
                "type": "rich_text",
                "block_id": "AgaY",
                "elements": [
                    {
                        "type": "rich_text_section",
                        "elements": [
                            {
                                "type": "text",
                                "text": "yeah this is a bad one, tbf its been a while"
                            }
                        ]
                    }
                ]
            }
        ]
    },
    {
        "user": "U7E6NE1DM",
        "type": "message",
        "ts": "1591882334.068600",
        "client_msg_id": "dccac5b5-5c23-4010-b2fc-f2819a3ef6e8",
        "text": "do you know in this case if they cancel the bets on a market or keep the bets?",
        "team": "T4G9NBD2M",
        "user_team": "T4G9NBD2M",
        "source_team": "T4G9NBD2M",
        "user_profile": {
            "avatar_hash": "g46d6d643667",
            "image_72": "https:\/\/secure.gravatar.com\/avatar\/446d6d643667e10a68a30e43be0d5cbf.jpg?s=72&d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0022-72.png",
            "first_name": "",
            "real_name": "Julio",
            "display_name": "",
            "team": "T4G9NBD2M",
            "name": "julien.wagener",
            "is_restricted": false,
            "is_ultra_restricted": false
        },
        "blocks": [
            {
                "type": "rich_text",
                "block_id": "SYuuk",
                "elements": [
                    {
                        "type": "rich_text_section",
                        "elements": [
                            {
                                "type": "text",
                                "text": "do you know in this case if they cancel the bets on a market or keep the bets?"
                            }
                        ]
                    }
                ]
            }
        ]
    },
    {
        "user": "ULDAVFDRP",
        "type": "message",
        "ts": "1591882427.068800",
        "client_msg_id": "1e49b11a-70a2-45da-987a-f94b48b42f1c",
        "text": "they persist",
        "team": "T4G9NBD2M",
        "user_team": "T4G9NBD2M",
        "source_team": "T4G9NBD2M",
        "user_profile": {
            "avatar_hash": "g46fe1c1bf03",
            "image_72": "https:\/\/secure.gravatar.com\/avatar\/b46fe1c1bf03e18d951e7032fb494283.jpg?s=72&d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0019-72.png",
            "first_name": "",
            "real_name": "Oliver Varney",
            "display_name": "Oliver Varney",
            "team": "T4G9NBD2M",
            "name": "oliverashleyvarney",
            "is_restricted": false,
            "is_ultra_restricted": false
        },
        "blocks": [
            {
                "type": "rich_text",
                "block_id": "xFAX",
                "elements": [
                    {
                        "type": "rich_text_section",
                        "elements": [
                            {
                                "type": "text",
                                "text": "they persist"
                            }
                        ]
                    }
                ]
            }
        ]
    },
    {
        "user": "ULDAVFDRP",
        "type": "message",
        "ts": "1591882443.069200",
        "client_msg_id": "94d2bc86-107e-4293-ab3e-16fdc5b03589",
        "text": "so offload on another exchange if you can",
        "team": "T4G9NBD2M",
        "user_team": "T4G9NBD2M",
        "source_team": "T4G9NBD2M",
        "user_profile": {
            "avatar_hash": "g46fe1c1bf03",
            "image_72": "https:\/\/secure.gravatar.com\/avatar\/b46fe1c1bf03e18d951e7032fb494283.jpg?s=72&d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0019-72.png",
            "first_name": "",
            "real_name": "Oliver Varney",
            "display_name": "Oliver Varney",
            "team": "T4G9NBD2M",
            "name": "oliverashleyvarney",
            "is_restricted": false,
            "is_ultra_restricted": false
        },
        "blocks": [
            {
                "type": "rich_text",
                "block_id": "TJ=s",
                "elements": [
                    {
                        "type": "rich_text_section",
                        "elements": [
                            {
                                "type": "text",
                                "text": "so offload on another exchange if you can"
                            }
                        ]
                    }
                ]
            }
        ]
    },
    {
        "user": "U7E6NE1DM",
        "type": "message",
        "ts": "1591882457.069400",
        "client_msg_id": "aae2aaa6-d8fb-465a-bd59-ae91e773a96c",
        "text": "poor scalpers then...",
        "team": "T4G9NBD2M",
        "user_team": "T4G9NBD2M",
        "source_team": "T4G9NBD2M",
        "user_profile": {
            "avatar_hash": "g46d6d643667",
            "image_72": "https:\/\/secure.gravatar.com\/avatar\/446d6d643667e10a68a30e43be0d5cbf.jpg?s=72&d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0022-72.png",
            "first_name": "",
            "real_name": "Julio",
            "display_name": "",
            "team": "T4G9NBD2M",
            "name": "julien.wagener",
            "is_restricted": false,
            "is_ultra_restricted": false
        },
        "blocks": [
            {
                "type": "rich_text",
                "block_id": "nG4Q",
                "elements": [
                    {
                        "type": "rich_text_section",
                        "elements": [
                            {
                                "type": "text",
                                "text": "poor scalpers then..."
                            }
                        ]
                    }
                ]
            }
        ]
    },
    {
        "user": "U7E6NE1DM",
        "type": "message",
        "ts": "1591882488.070000",
        "client_msg_id": "36d78dc2-6c79-4488-8682-3a123ee47b94",
        "text": "thanks i do not have open orders, it was just for curiosity",
        "team": "T4G9NBD2M",
        "user_team": "T4G9NBD2M",
        "source_team": "T4G9NBD2M",
        "user_profile": {
            "avatar_hash": "g46d6d643667",
            "image_72": "https:\/\/secure.gravatar.com\/avatar\/446d6d643667e10a68a30e43be0d5cbf.jpg?s=72&d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0022-72.png",
            "first_name": "",
            "real_name": "Julio",
            "display_name": "",
            "team": "T4G9NBD2M",
            "name": "julien.wagener",
            "is_restricted": false,
            "is_ultra_restricted": false
        },
        "blocks": [
            {
                "type": "rich_text",
                "block_id": "WiUM",
                "elements": [
                    {
                        "type": "rich_text_section",
                        "elements": [
                            {
                                "type": "text",
                                "text": "thanks i do not have open orders, it was just for curiosity"
                            }
                        ]
                    }
                ]
            }
        ]
    },
    {
        "user": "UU1URJ8L8",
        "type": "message",
        "ts": "1591883309.070300",
        "client_msg_id": "33589314-7ee0-4d18-b144-b78d9afa2840",
        "text": "`\"errorCode\":\"ERROR_IN_MATCHER\"`",
        "team": "T4G9NBD2M",
        "user_team": "T4G9NBD2M",
        "source_team": "T4G9NBD2M",
        "user_profile": {
            "avatar_hash": "g7a5a7d77117",
            "image_72": "https:\/\/secure.gravatar.com\/avatar\/7a5a7d7711746fb16f400e11af544b2e.jpg?s=72&d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0012-72.png",
            "first_name": "",
            "real_name": "Remi",
            "display_name": "Remi",
            "team": "T4G9NBD2M",
            "name": "services",
            "is_restricted": false,
            "is_ultra_restricted": false
        },
        "blocks": [
            {
                "type": "rich_text",
                "block_id": "uVU",
                "elements": [
                    {
                        "type": "rich_text_section",
                        "elements": [
                            {
                                "type": "text",
                                "text": "\"errorCode\":\"ERROR_IN_MATCHER\"",
                                "style": {
                                    "code": true
                                }
                            }
                        ]
                    }
                ]
            }
        ]
    },
    {
        "user": "U4H19D1D2",
        "type": "message",
        "ts": "1591884622.070900",
        "client_msg_id": "004290e4-529d-4aad-9d03-ad90a2e3c024",
        "text": "Anyone know how to get the error when this happens? (docker\/python)\n```14:03:07\nSegmentation fault (core dumped)```",
        "team": "T4G9NBD2M",
        "user_team": "T4G9NBD2M",
        "source_team": "T4G9NBD2M",
        "user_profile": {
            "avatar_hash": "137c5a3ef323",
            "image_72": "https:\/\/avatars.slack-edge.com\/2024-03-15\/6819395173841_137c5a3ef323f1944a1a_72.png",
            "first_name": "liam",
            "real_name": "liam",
            "display_name": "liam",
            "team": "T4G9NBD2M",
            "name": "liam",
            "is_restricted": false,
            "is_ultra_restricted": false
        },
        "blocks": [
            {
                "type": "rich_text",
                "block_id": "Jz9s",
                "elements": [
                    {
                        "type": "rich_text_section",
                        "elements": [
                            {
                                "type": "text",
                                "text": "Anyone know how to get the error when this happens? (docker\/python)\n"
                            }
                        ]
                    },
                    {
                        "type": "rich_text_preformatted",
                        "elements": [
                            {
                                "type": "text",
                                "text": "14:03:07\nSegmentation fault (core dumped)"
                            }
                        ]
                    }
                ]
            }
        ]
    },
    {
        "user": "UUCD6P13J",
        "type": "message",
        "ts": "1591884646.071300",
        "client_msg_id": "672bd338-c44d-4493-84c5-f311e81100c1",
        "text": "i think it usually dumps a file to disk",
        "team": "T4G9NBD2M",
        "user_team": "T4G9NBD2M",
        "source_team": "T4G9NBD2M",
        "user_profile": {
            "avatar_hash": "g6898272ab81",
            "image_72": "https:\/\/secure.gravatar.com\/avatar\/6898272ab810e419c2c9ca9af1e53da8.jpg?s=72&d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0012-72.png",
            "first_name": "Lee",
            "real_name": "Lee U",
            "display_name": "Lee",
            "team": "T4G9NBD2M",
            "name": "lunswor",
            "is_restricted": false,
            "is_ultra_restricted": false
        },
        "blocks": [
            {
                "type": "rich_text",
                "block_id": "g8HRN",
                "elements": [
                    {
                        "type": "rich_text_section",
                        "elements": [
                            {
                                "type": "text",
                                "text": "i think it usually dumps a file to disk"
                            }
                        ]
                    }
                ]
            }
        ]
    },
    {
        "user": "U4H19D1D2",
        "type": "message",
        "ts": "1591884933.071900",
        "client_msg_id": "307dd45a-4baa-4018-8e69-9f2e43183b69",
        "text": "nice, found it, ujson causing issues, anyone able to translate?\n```Jun 11 14:03:07 ip-172-31-38-187 kernel: python[5081]: segfault at 0 ip 00007f4b75638a49 sp 00007f4b6cff76f0 error 4 in ujson.cpython-38-x86_64-linux-gnu.so[7f4b7562f000+14000]```",
        "team": "T4G9NBD2M",
        "user_team": "T4G9NBD2M",
        "source_team": "T4G9NBD2M",
        "user_profile": {
            "avatar_hash": "137c5a3ef323",
            "image_72": "https:\/\/avatars.slack-edge.com\/2024-03-15\/6819395173841_137c5a3ef323f1944a1a_72.png",
            "first_name": "liam",
            "real_name": "liam",
            "display_name": "liam",
            "team": "T4G9NBD2M",
            "name": "liam",
            "is_restricted": false,
            "is_ultra_restricted": false
        },
        "blocks": [
            {
                "type": "rich_text",
                "block_id": "pPx",
                "elements": [
                    {
                        "type": "rich_text_section",
                        "elements": [
                            {
                                "type": "text",
                                "text": "nice, found it, ujson causing issues, anyone able to translate?\n"
                            }
                        ]
                    },
                    {
                        "type": "rich_text_preformatted",
                        "elements": [
                            {
                                "type": "text",
                                "text": "Jun 11 14:03:07 ip-172-31-38-187 kernel: python[5081]: segfault at 0 ip 00007f4b75638a49 sp 00007f4b6cff76f0 error 4 in ujson.cpython-38-x86_64-linux-gnu.so[7f4b7562f000+14000]"
                            }
                        ]
                    }
                ]
            }
        ]
    },
    {
        "user": "UBS7QANF3",
        "type": "message",
        "ts": "1591884994.073000",
        "edited": {
            "user": "UBS7QANF3",
            "ts": "1591885003.000000"
        },
        "client_msg_id": "B6FD0FF3-837A-4CED-B70F-606701F8C032",
        "text": "You’d need to have compiled with debug symbols to see the source file name and line number ",
        "team": "T4G9NBD2M",
        "user_team": "T4G9NBD2M",
        "source_team": "T4G9NBD2M",
        "user_profile": {
            "avatar_hash": "gaaf844a4a90",
            "image_72": "https:\/\/secure.gravatar.com\/avatar\/eaaf844a4a905431d83430e563b077aa.jpg?s=72&d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0011-72.png",
            "first_name": "",
            "real_name": "Maurice Berk",
            "display_name": "Mo",
            "team": "T4G9NBD2M",
            "name": "maurice",
            "is_restricted": false,
            "is_ultra_restricted": false
        },
        "blocks": [
            {
                "type": "rich_text",
                "block_id": "58aJ",
                "elements": [
                    {
                        "type": "rich_text_section",
                        "elements": [
                            {
                                "type": "text",
                                "text": "You’d need to have compiled with debug symbols to see the source file name and line number "
                            }
                        ]
                    }
                ]
            }
        ]
    },
    {
        "user": "UBS7QANF3",
        "type": "message",
        "ts": "1591885142.073400",
        "client_msg_id": "e1999eb0-3ca8-4616-a52f-63778f70c52f",
        "text": "In other words, can't really tell anything from that",
        "team": "T4G9NBD2M",
        "user_team": "T4G9NBD2M",
        "source_team": "T4G9NBD2M",
        "user_profile": {
            "avatar_hash": "gaaf844a4a90",
            "image_72": "https:\/\/secure.gravatar.com\/avatar\/eaaf844a4a905431d83430e563b077aa.jpg?s=72&d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0011-72.png",
            "first_name": "",
            "real_name": "Maurice Berk",
            "display_name": "Mo",
            "team": "T4G9NBD2M",
            "name": "maurice",
            "is_restricted": false,
            "is_ultra_restricted": false
        },
        "blocks": [
            {
                "type": "rich_text",
                "block_id": "dSU",
                "elements": [
                    {
                        "type": "rich_text_section",
                        "elements": [
                            {
                                "type": "text",
                                "text": "In other words, can't really tell anything from that"
                            }
                        ]
                    }
                ]
            }
        ]
    },
    {
        "user": "U4H19D1D2",
        "type": "message",
        "ts": "1591885780.074100",
        "client_msg_id": "b49ff684-cd88-4cc5-a0c8-de712bce5fd5",
        "text": "yeah, looking at the issue log it could be one of many different things, wonder if bflw should using a different library..",
        "team": "T4G9NBD2M",
        "user_team": "T4G9NBD2M",
        "source_team": "T4G9NBD2M",
        "user_profile": {
            "avatar_hash": "137c5a3ef323",
            "image_72": "https:\/\/avatars.slack-edge.com\/2024-03-15\/6819395173841_137c5a3ef323f1944a1a_72.png",
            "first_name": "liam",
            "real_name": "liam",
            "display_name": "liam",
            "team": "T4G9NBD2M",
            "name": "liam",
            "is_restricted": false,
            "is_ultra_restricted": false
        },
        "blocks": [
            {
                "type": "rich_text",
                "block_id": "uUyo",
                "elements": [
                    {
                        "type": "rich_text_section",
                        "elements": [
                            {
                                "type": "text",
                                "text": "yeah, looking at the issue log it could be one of many different things, wonder if bflw should using a different library.."
                            }
                        ]
                    }
                ]
            }
        ]
    },
    {
        "user": "U92CASP1B",
        "type": "message",
        "ts": "1591885954.076300",
        "client_msg_id": "d5a9358b-7696-47bf-9f6f-096b1b17e14f",
        "text": "I have made the decision to use the built in library for \"live\" data, for historical I'm more open to unstable libraries for speed.",
        "team": "T4G9NBD2M",
        "user_team": "T4G9NBD2M",
        "source_team": "T4G9NBD2M",
        "user_profile": {
            "avatar_hash": "g374352c822f",
            "image_72": "https:\/\/secure.gravatar.com\/avatar\/a374352c822fa856c4cfdd782bbb8669.jpg?s=72&d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0007-72.png",
            "first_name": "Jonatan",
            "real_name": "Jonatan",
            "display_name": "Jonatan (skyw)",
            "team": "T4G9NBD2M",
            "name": "almen.jonatan",
            "is_restricted": false,
            "is_ultra_restricted": false
        },
        "blocks": [
            {
                "type": "rich_text",
                "block_id": "tfG",
                "elements": [
                    {
                        "type": "rich_text_section",
                        "elements": [
                            {
                                "type": "text",
                                "text": "I have made the decision to use the built in library for \"live\" data, for historical I'm more open to unstable libraries for speed."
                            }
                        ]
                    }
                ]
            }
        ],
        "reactions": [
            {
                "name": "thinking_face",
                "users": [
                    "U4H19D1D2"
                ],
                "count": 1
            }
        ]
    },
    {
        "user": "UBS7QANF3",
        "type": "message",
        "ts": "1591886529.077800",
        "client_msg_id": "0d3f6da4-c962-49fb-9343-1829c820c212",
        "text": "Can't say I've ever had any problems with it. Seems likely the issue here was it was given junk data because of Betfair API issues; obviously it would be better if it could handle this more robustly but if the two are related it's not like you'd have been able to run your system anyway",
        "team": "T4G9NBD2M",
        "user_team": "T4G9NBD2M",
        "source_team": "T4G9NBD2M",
        "user_profile": {
            "avatar_hash": "gaaf844a4a90",
            "image_72": "https:\/\/secure.gravatar.com\/avatar\/eaaf844a4a905431d83430e563b077aa.jpg?s=72&d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0011-72.png",
            "first_name": "",
            "real_name": "Maurice Berk",
            "display_name": "Mo",
            "team": "T4G9NBD2M",
            "name": "maurice",
            "is_restricted": false,
            "is_ultra_restricted": false
        },
        "blocks": [
            {
                "type": "rich_text",
                "block_id": "0pn",
                "elements": [
                    {
                        "type": "rich_text_section",
                        "elements": [
                            {
                                "type": "text",
                                "text": "Can't say I've ever had any problems with it. Seems likely the issue here was it was given junk data because of Betfair API issues; obviously it would be better if it could handle this more robustly but if the two are related it's not like you'd have been able to run your system anyway"
                            }
                        ]
                    }
                ]
            }
        ]
    },
    {
        "user": "U4H19D1D2",
        "type": "message",
        "ts": "1591888488.078600",
        "client_msg_id": "b36a5f48-7e09-49eb-bf6c-b3fa13f901e3",
        "text": "Yeah I think your right, just hate the idea of an error like this",
        "team": "T4G9NBD2M",
        "user_team": "T4G9NBD2M",
        "source_team": "T4G9NBD2M",
        "user_profile": {
            "avatar_hash": "137c5a3ef323",
            "image_72": "https:\/\/avatars.slack-edge.com\/2024-03-15\/6819395173841_137c5a3ef323f1944a1a_72.png",
            "first_name": "liam",
            "real_name": "liam",
            "display_name": "liam",
            "team": "T4G9NBD2M",
            "name": "liam",
            "is_restricted": false,
            "is_ultra_restricted": false
        },
        "blocks": [
            {
                "type": "rich_text",
                "block_id": "1pGd",
                "elements": [
                    {
                        "type": "rich_text_section",
                        "elements": [
                            {
                                "type": "text",
                                "text": "Yeah I think your right, just hate the idea of an error like this"
                            }
                        ]
                    }
                ]
            }
        ]
    },
    {
        "user": "U010GM77S4W",
        "type": "message",
        "ts": "1591889558.079300",
        "client_msg_id": "7b3c81b4-b23d-4b46-b3a3-c21c846fde2c",
        "text": "Anyone have any error messages from the API around the exact time it crashed? Betfair claiming to me the crash was at 13:25 but it was clearly around 13:18",
        "team": "T4G9NBD2M",
        "user_team": "T4G9NBD2M",
        "source_team": "T4G9NBD2M",
        "user_profile": {
            "avatar_hash": "66cae8f5399c",
            "image_72": "https:\/\/avatars.slack-edge.com\/2020-03-20\/1004508356769_66cae8f5399cf91a07ae_72.jpg",
            "first_name": "",
            "real_name": "mandelbot",
            "display_name": "mandelbot",
            "team": "T4G9NBD2M",
            "name": "demhadbale",
            "is_restricted": false,
            "is_ultra_restricted": false
        },
        "blocks": [
            {
                "type": "rich_text",
                "block_id": "RomU",
                "elements": [
                    {
                        "type": "rich_text_section",
                        "elements": [
                            {
                                "type": "text",
                                "text": "Anyone have any error messages from the API around the exact time it crashed? Betfair claiming to me the crash was at 13:25 but it was clearly around 13:18"
                            }
                        ]
                    }
                ]
            }
        ]
    },
    {
        "user": "U4H19D1D2",
        "type": "message",
        "ts": "1591889796.079700",
        "client_msg_id": "ee0147d0-c558-4934-9e2b-8806c510a228",
        "text": "Streams went into 503 at around 13:22:06",
        "team": "T4G9NBD2M",
        "user_team": "T4G9NBD2M",
        "source_team": "T4G9NBD2M",
        "user_profile": {
            "avatar_hash": "137c5a3ef323",
            "image_72": "https:\/\/avatars.slack-edge.com\/2024-03-15\/6819395173841_137c5a3ef323f1944a1a_72.png",
            "first_name": "liam",
            "real_name": "liam",
            "display_name": "liam",
            "team": "T4G9NBD2M",
            "name": "liam",
            "is_restricted": false,
            "is_ultra_restricted": false
        },
        "blocks": [
            {
                "type": "rich_text",
                "block_id": "C2vf7",
                "elements": [
                    {
                        "type": "rich_text_section",
                        "elements": [
                            {
                                "type": "text",
                                "text": "Streams went into 503 at around 13:22:06"
                            }
                        ]
                    }
                ]
            }
        ]
    },
    {
        "user": "U4H19D1D2",
        "type": "message",
        "ts": "1591889942.080100",
        "client_msg_id": "8bca36a0-5cb3-4356-8c48-b24e514a3c5e",
        "text": "13:17:19 is when I started getting errors",
        "team": "T4G9NBD2M",
        "user_team": "T4G9NBD2M",
        "source_team": "T4G9NBD2M",
        "user_profile": {
            "avatar_hash": "137c5a3ef323",
            "image_72": "https:\/\/avatars.slack-edge.com\/2024-03-15\/6819395173841_137c5a3ef323f1944a1a_72.png",
            "first_name": "liam",
            "real_name": "liam",
            "display_name": "liam",
            "team": "T4G9NBD2M",
            "name": "liam",
            "is_restricted": false,
            "is_ultra_restricted": false
        },
        "blocks": [
            {
                "type": "rich_text",
                "block_id": "ddIHW",
                "elements": [
                    {
                        "type": "rich_text_section",
                        "elements": [
                            {
                                "type": "text",
                                "text": "13:17:19 is when I started getting errors"
                            }
                        ]
                    }
                ]
            }
        ]
    },
    {
        "user": "U010GM77S4W",
        "type": "message",
        "ts": "1591889980.080300",
        "client_msg_id": "1cf7529f-346d-437a-a58f-cdb723ad6fe3",
        "text": "can I have a log please?",
        "team": "T4G9NBD2M",
        "user_team": "T4G9NBD2M",
        "source_team": "T4G9NBD2M",
        "user_profile": {
            "avatar_hash": "66cae8f5399c",
            "image_72": "https:\/\/avatars.slack-edge.com\/2020-03-20\/1004508356769_66cae8f5399cf91a07ae_72.jpg",
            "first_name": "",
            "real_name": "mandelbot",
            "display_name": "mandelbot",
            "team": "T4G9NBD2M",
            "name": "demhadbale",
            "is_restricted": false,
            "is_ultra_restricted": false
        },
        "blocks": [
            {
                "type": "rich_text",
                "block_id": "2ZK8=",
                "elements": [
                    {
                        "type": "rich_text_section",
                        "elements": [
                            {
                                "type": "text",
                                "text": "can I have a log please?"
                            }
                        ]
                    }
                ]
            }
        ]
    },
    {
        "text": "```{\n    \"asctime\": \"2020-06-11 12:17:19,379\",\n    \"levelname\": \"ERROR\",\n    \"message\": \"_get_cleared_markets error\",\n    \"filename\": \"accounthandler.py\",\n    \"funcName\": \"_get_cleared_markets\",\n    \"module\": \"accounthandler\",\n    \"process\": 6,\n    \"threadName\": \"account_handler\",\n    \"exc_info\": \"Traceback (most recent call last):\\n  File \\\"\/usr\/local\/lib\/python3.8\/site-packages\/requests\/packages\/urllib3\/connectionpool.py\\\", line 597, in urlopen\\n    httplib_response = self._make_request(conn, method, url,\\n  File \\\"\/usr\/local\/lib\/python3.8\/site-packages\/requests\/packages\/urllib3\/connectionpool.py\\\", line 386, in _make_request\\n    six.raise_from(e, None)\\n  File \\\"<string>\\\", line 2, in raise_from\\n  File \\\"\/usr\/local\/lib\/python3.8\/site-packages\/requests\/packages\/urllib3\/connectionpool.py\\\", line 382, in _make_request\\n    httplib_response = conn.getresponse()\\n  File \\\"\/usr\/local\/lib\/python3.8\/site-packages\/sentry_sdk\/integrations\/stdlib.py\\\", line 102, in getresponse\\n    rv = real_getresponse(self, *args, **kwargs)\\n  File \\\"\/usr\/local\/lib\/python3.8\/http\/client.py\\\", line 1332, in getresponse\\n    response.begin()\\n  File \\\"\/usr\/local\/lib\/python3.8\/http\/client.py\\\", line 303, in begin\\n    version, status, reason = self._read_status()\\n  File \\\"\/usr\/local\/lib\/python3.8\/http\/client.py\\\", line 264, in _read_status\\n    line = str(self.fp.readline(_MAXLINE + 1), \\\"iso-8859-1\\\")\\n  File \\\"\/usr\/local\/lib\/python3.8\/socket.py\\\", line 669, in readinto\\n    return self._sock.recv_into(b)\\n  File \\\"\/usr\/local\/lib\/python3.8\/ssl.py\\\", line 1241, in recv_into\\n    return self.read(nbytes, buffer)\\n  File \\\"\/usr\/local\/lib\/python3.8\/ssl.py\\\", line 1099, in read\\n    return self._sslobj.read(len, buffer)\\nConnectionResetError: [Errno 104] Connection reset by peer\\n\\nDuring handling of the above exception, another exception occurred:\\n\\nTraceback (most recent call last):\\n  File \\\"\/usr\/local\/lib\/python3.8\/site-packages\/requests\/adapters.py\\\", line 413, in send\\n    resp = conn.urlopen(\\n  File \\\"\/usr\/local\/lib\/python3.8\/site-packages\/requests\/packages\/urllib3\/connectionpool.py\\\", line 648, in urlopen\\n    retries = retries.increment(method, url, error=e, _pool=self,\\n  File \\\"\/usr\/local\/lib\/python3.8\/site-packages\/requests\/packages\/urllib3\/util\/retry.py\\\", line 347, in increment\\n    raise six.reraise(type(error), error, _stacktrace)\\n  File \\\"\/usr\/local\/lib\/python3.8\/site-packages\/requests\/packages\/urllib3\/packages\/six.py\\\", line 685, in reraise\\n    raise value.with_traceback(tb)\\n  File \\\"\/usr\/local\/lib\/python3.8\/site-packages\/requests\/packages\/urllib3\/connectionpool.py\\\", line 597, in urlopen\\n    httplib_response = self._make_request(conn, method, url,\\n  File \\\"\/usr\/local\/lib\/python3.8\/site-packages\/requests\/packages\/urllib3\/connectionpool.py\\\", line 386, in _make_request\\n    six.raise_from(e, None)\\n  File \\\"<string>\\\", line 2, in raise_from\\n  File \\\"\/usr\/local\/lib\/python3.8\/site-packages\/requests\/packages\/urllib3\/connectionpool.py\\\", line 382, in _make_request\\n    httplib_response = conn.getresponse()\\n  File \\\"\/usr\/local\/lib\/python3.8\/site-packages\/sentry_sdk\/integrations\/stdlib.py\\\", line 102, in getresponse\\n    rv = real_getresponse(self, *args, **kwargs)\\n  File \\\"\/usr\/local\/lib\/python3.8\/http\/client.py\\\", line 1332, in getresponse\\n    response.begin()\\n  File \\\"\/usr\/local\/lib\/python3.8\/http\/client.py\\\", line 303, in begin\\n    version, status, reason = self._read_status()\\n  File \\\"\/usr\/local\/lib\/python3.8\/http\/client.py\\\", line 264, in _read_status\\n    line = str(self.fp.readline(_MAXLINE + 1), \\\"iso-8859-1\\\")\\n  File \\\"\/usr\/local\/lib\/python3.8\/socket.py\\\", line 669, in readinto\\n    return self._sock.recv_into(b)\\n  File \\\"\/usr\/local\/lib\/python3.8\/ssl.py\\\", line 1241, in recv_into\\n    return self.read(nbytes, buffer)\\n  File \\\"\/usr\/local\/lib\/python3.8\/ssl.py\\\", line 1099, in read\\n    return self._sslobj.read(len, buffer)\\nrequests.packages.urllib3.exceptions.ProtocolError: ('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))\\n\\nDuring handling of the above exception, another exception occurred:\\n\\nTraceback (most recent call last):\\n  File \\\"\/usr\/local\/lib\/python3.8\/site-packages\/betfairlightweight-0.0.0b13-py3.8.egg\/betfairlightweight\/endpoints\/baseendpoint.py\\\", line 36, in request\\n    response = session.post(\\n  File \\\"\/usr\/local\/lib\/python3.8\/site-packages\/requests\/api.py\\\", line 110, in post\\n    return request('post', url, data=data, json=json, **kwargs)\\n  File \\\"\/usr\/local\/lib\/python3.8\/site-packages\/requests\/api.py\\\", line 56, in request\\n    return session.request(method=method, url=url, **kwargs)\\n  File \\\"\/usr\/local\/lib\/python3.8\/site-packages\/requests\/sessions.py\\\", line 488, in request\\n    resp = self.send(prep, **send_kwargs)\\n  File \\\"\/usr\/local\/lib\/python3.8\/site-packages\/requests\/sessions.py\\\", line 609, in send\\n    r = adapter.send(request, **kwargs)\\n  File \\\"\/usr\/local\/lib\/python3.8\/site-packages\/requests\/adapters.py\\\", line 473, in send\\n    raise ConnectionError(err, request=request)\\nrequests.exceptions.ConnectionError: ('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))\\n\\nDuring handling of the above exception, another exception occurred:\\n\\nTraceback (most recent call last):\\n  File \\\"\/usr\/local\/lib\/python3.8\/site-packages\/equine-1.11.2-py3.8.egg\/equine\/account\/accounthandler.py\\\", line 130, in _get_cleared_markets\\n    cleared_markets = client.betting_client.betting.list_cleared_orders(\\n  File \\\"\/usr\/local\/lib\/python3.8\/site-packages\/betfairlightweight-0.0.0b13-py3.8.egg\/betfairlightweight\/endpoints\/betting.py\\\", line 438, in list_cleared_orders\\n    (response, response_json, elapsed_time) = self.request(method, params, session)\\n  File \\\"\/usr\/local\/lib\/python3.8\/site-packages\/betfairlightweight-0.0.0b13-py3.8.egg\/betfairlightweight\/endpoints\/baseendpoint.py\\\", line 43, in request\\n    raise APIError(None, method, params, e)\\nbetfairlightweight.exceptions.APIError: SportsAPING\/v1.0\/listClearedOrders \\nParams: {'betStatus': 'SETTLED', 'marketIds': ['1.170718126'], 'settledDateRange': {'from': None, 'to': None}, 'groupBy': 'MARKET'} \\nException: ('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))\",\n    \"trading_function\": \"list_cleared_orders\",\n    \"response\": \"SportsAPING\/v1.0\/listClearedOrders \\nParams: {'betStatus': 'SETTLED', 'marketIds': ['1.170718126'], 'settledDateRange': {'from': None, 'to': None}, 'groupBy': 'MARKET'} \\nException: ('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))\"\n}```",
        "files": [
            {
                "id": "F015YK0NHJL",
                "mode": "hidden_by_limit"
            }
        ],
        "upload": false,
        "user": "U4H19D1D2",
        "display_as_bot": false,
        "blocks": [
            {
                "type": "rich_text",
                "block_id": "nPa",
                "elements": [
                    {
                        "type": "rich_text_preformatted",
                        "elements": [
                            {
                                "type": "text",
                                "text": "{\n    \"asctime\": \"2020-06-11 12:17:19,379\",\n    \"levelname\": \"ERROR\",\n    \"message\": \"_get_cleared_markets error\",\n    \"filename\": \"accounthandler.py\",\n    \"funcName\": \"_get_cleared_markets\",\n    \"module\": \"accounthandler\",\n    \"process\": 6,\n    \"threadName\": \"account_handler\",\n    \"exc_info\": \"Traceback (most recent call last):\\n  File \\\"\/usr\/local\/lib\/python3.8\/site-packages\/requests\/packages\/urllib3\/connectionpool.py\\\", line 597, in urlopen\\n    httplib_response = self._make_request(conn, method, url,\\n  File \\\"\/usr\/local\/lib\/python3.8\/site-packages\/requests\/packages\/urllib3\/connectionpool.py\\\", line 386, in _make_request\\n    six.raise_from(e, None)\\n  File \\\"<string>\\\", line 2, in raise_from\\n  File \\\"\/usr\/local\/lib\/python3.8\/site-packages\/requests\/packages\/urllib3\/connectionpool.py\\\", line 382, in _make_request\\n    httplib_response = conn.getresponse()\\n  File \\\"\/usr\/local\/lib\/python3.8\/site-packages\/sentry_sdk\/integrations\/stdlib.py\\\", line 102, in getresponse\\n    rv = real_getresponse(self, *args, **kwargs)\\n  File \\\"\/usr\/local\/lib\/python3.8\/http\/client.py\\\", line 1332, in getresponse\\n    response.begin()\\n  File \\\"\/usr\/local\/lib\/python3.8\/http\/client.py\\\", line 303, in begin\\n    version, status, reason = self._read_status()\\n  File \\\"\/usr\/local\/lib\/python3.8\/http\/client.py\\\", line 264, in _read_status\\n    line = str(self.fp.readline(_MAXLINE + 1), \\\"iso-8859-1\\\")\\n  File \\\"\/usr\/local\/lib\/python3.8\/socket.py\\\", line 669, in readinto\\n    return self._sock.recv_into(b)\\n  File \\\"\/usr\/local\/lib\/python3.8\/ssl.py\\\", line 1241, in recv_into\\n    return self.read(nbytes, buffer)\\n  File \\\"\/usr\/local\/lib\/python3.8\/ssl.py\\\", line 1099, in read\\n    return self._sslobj.read(len, buffer)\\nConnectionResetError: [Errno 104] Connection reset by peer\\n\\nDuring handling of the above exception, another exception occurred:\\n\\nTraceback (most recent call last):\\n  File \\\"\/usr\/local\/lib\/python3.8\/site-packages\/requests\/adapters.py\\\", line 413, in send\\n    resp = conn.urlopen(\\n  File \\\"\/usr\/local\/lib\/python3.8\/site-packages\/requests\/packages\/urllib3\/connectionpool.py\\\", line 648, in urlopen\\n    retries = retries.increment(method, url, error=e, _pool=self,\\n  File \\\"\/usr\/local\/lib\/python3.8\/site-packages\/requests\/packages\/urllib3\/util\/retry.py\\\", line 347, in increment\\n    raise six.reraise(type(error), error, _stacktrace)\\n  File \\\"\/usr\/local\/lib\/python3.8\/site-packages\/requests\/packages\/urllib3\/packages\/six.py\\\", line 685, in reraise\\n    raise value.with_traceback(tb)\\n  File \\\"\/usr\/local\/lib\/python3.8\/site-packages\/requests\/packages\/urllib3\/connectionpool.py\\\", line 597, in urlopen\\n    httplib_response = self._make_request(conn, method, url,\\n  File \\\"\/usr\/local\/lib\/python3.8\/site-packages\/requests\/packages\/urllib3\/connectionpool.py\\\", line 386, in _make_request\\n    six.raise_from(e, None)\\n  File \\\"<string>\\\", line 2, in raise_from\\n  File \\\"\/usr\/local\/lib\/python3.8\/site-packages\/requests\/packages\/urllib3\/connectionpool.py\\\", line 382, in _make_request\\n    httplib_response = conn.getresponse()\\n  File \\\"\/usr\/local\/lib\/python3.8\/site-packages\/sentry_sdk\/integrations\/stdlib.py\\\", line 102, in getresponse\\n    rv = real_getresponse(self, *args, **kwargs)\\n  File \\\"\/usr\/local\/lib\/python3.8\/http\/client.py\\\", line 1332, in getresponse\\n    response.begin()\\n  File \\\"\/usr\/local\/lib\/python3.8\/http\/client.py\\\", line 303, in begin\\n    version, status, reason = self._read_status()\\n  File \\\"\/usr\/local\/lib\/python3.8\/http\/client.py\\\", line 264, in _read_status\\n    line = str(self.fp.readline(_MAXLINE + 1), \\\"iso-8859-1\\\")\\n  File \\\"\/usr\/local\/lib\/python3.8\/socket.py\\\", line 669, in readinto\\n    return self._sock.recv_into(b)\\n  File \\\"\/usr\/local\/lib\/python3.8\/ssl.py\\\", line 1241, in recv_into\\n    return self.read(nbytes, buffer)\\n  File \\\"\/usr\/local\/lib\/python3.8\/ssl.py\\\", line 1099, in read\\n    return self._sslobj.read(len, buffer)\\nrequests.packages.urllib3.exceptions.ProtocolError: ('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))\\n\\nDuring handling of the above exception, another exception occurred:\\n\\nTraceback (most recent call last):\\n  File \\\"\/usr\/local\/lib\/python3.8\/site-packages\/betfairlightweight-0.0.0b13-py3.8.egg\/betfairlightweight\/endpoints\/baseendpoint.py\\\", line 36, in request\\n    response = session.post(\\n  File \\\"\/usr\/local\/lib\/python3.8\/site-packages\/requests\/api.py\\\", line 110, in post\\n    return request('post', url, data=data, json=json, **kwargs)\\n  File \\\"\/usr\/local\/lib\/python3.8\/site-packages\/requests\/api.py\\\", line 56, in request\\n    return session.request(method=method, url=url, **kwargs)\\n  File \\\"\/usr\/local\/lib\/python3.8\/site-packages\/requests\/sessions.py\\\", line 488, in request\\n    resp = self.send(prep, **send_kwargs)\\n  File \\\"\/usr\/local\/lib\/python3.8\/site-packages\/requests\/sessions.py\\\", line 609, in send\\n    r = adapter.send(request, **kwargs)\\n  File \\\"\/usr\/local\/lib\/python3.8\/site-packages\/requests\/adapters.py\\\", line 473, in send\\n    raise ConnectionError(err, request=request)\\nrequests.exceptions.ConnectionError: ('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))\\n\\nDuring handling of the above exception, another exception occurred:\\n\\nTraceback (most recent call last):\\n  File \\\"\/usr\/local\/lib\/python3.8\/site-packages\/equine-1.11.2-py3.8.egg\/equine\/account\/accounthandler.py\\\", line 130, in _get_cleared_markets\\n    cleared_markets = client.betting_client.betting.list_cleared_orders(\\n  File \\\"\/usr\/local\/lib\/python3.8\/site-packages\/betfairlightweight-0.0.0b13-py3.8.egg\/betfairlightweight\/endpoints\/betting.py\\\", line 438, in list_cleared_orders\\n    (response, response_json, elapsed_time) = self.request(method, params, session)\\n  File \\\"\/usr\/local\/lib\/python3.8\/site-packages\/betfairlightweight-0.0.0b13-py3.8.egg\/betfairlightweight\/endpoints\/baseendpoint.py\\\", line 43, in request\\n    raise APIError(None, method, params, e)\\nbetfairlightweight.exceptions.APIError: SportsAPING\/v1.0\/listClearedOrders \\nParams: {'betStatus': 'SETTLED', 'marketIds': ['1.170718126'], 'settledDateRange': {'from': None, 'to': None}, 'groupBy': 'MARKET'} \\nException: ('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))\",\n    \"trading_function\": \"list_cleared_orders\",\n    \"response\": \"SportsAPING\/v1.0\/listClearedOrders \\nParams: {'betStatus': 'SETTLED', 'marketIds': ['1.170718126'], 'settledDateRange': {'from': None, 'to': None}, 'groupBy': 'MARKET'} \\nException: ('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))\"\n}"
                            }
                        ]
                    }
                ]
            }
        ],
        "type": "message",
        "ts": "1591890255.080500"
    },
    {
        "user": "U010GM77S4W",
        "type": "message",
        "ts": "1591890289.080900",
        "client_msg_id": "ba2f5fb3-996b-461c-8374-ea2cdfd7fffa",
        "text": "thanks!",
        "team": "T4G9NBD2M",
        "user_team": "T4G9NBD2M",
        "source_team": "T4G9NBD2M",
        "user_profile": {
            "avatar_hash": "66cae8f5399c",
            "image_72": "https:\/\/avatars.slack-edge.com\/2020-03-20\/1004508356769_66cae8f5399cf91a07ae_72.jpg",
            "first_name": "",
            "real_name": "mandelbot",
            "display_name": "mandelbot",
            "team": "T4G9NBD2M",
            "name": "demhadbale",
            "is_restricted": false,
            "is_ultra_restricted": false
        },
        "blocks": [
            {
                "type": "rich_text",
                "block_id": "4=\/o",
                "elements": [
                    {
                        "type": "rich_text_section",
                        "elements": [
                            {
                                "type": "text",
                                "text": "thanks!"
                            }
                        ]
                    }
                ]
            }
        ]
    },
    {
        "text": "",
        "files": [
            {
                "id": "F015YKCAS0G",
                "mode": "hidden_by_limit"
            }
        ],
        "upload": false,
        "user": "U4H19D1D2",
        "display_as_bot": false,
        "type": "message",
        "ts": "1591890336.081000"
    }
]